### ISSUE TITLE
Update Editor Guidelines: DOI workflow

### ISSUE THREAD

**Issue by rivaquiroga** [programminghistorian/jekyll#1776]
As discussed in https://github.com/programminghistorian/jekyll/issues/1684, Editor Guidelines should be updated to include the DOI request workflow. 
This will only affect the ME checklist section. 

@spapastamkou, @svmelton, I will write a draft, and reach you back for your review

**Comment by walshbr** [comment]
I might also change the text of the preceding Twitter bot section of the guidelines from 

"link (column C) - the link to the lesson."

to

"link (column C) - the DOI for the lesson."

since we're now using the DOI's in the bot.

**Comment by rivaquiroga** [comment]
@spapastamkou, @svmelton, here is a proposal for the new step of the checklist. The three main points are: 1) be sure that everything is ok before requesting the DOI, 2) link to the stepts for requesting the DOI on the wiki,  3) how long this part of the process might take. 
First I thought this should number 5 in our checklist, but now I realized that without the DOI the builds fails, so maybe is better to put it before "Confirm all links and YAML headers are functioning correctly"
@spapastamkou, you just went through the process. Is there anything else you think we might add or clarify?

----

## 4) Request DOI

Once you are sure that the submission meets all our publication guidelines and is ready to be published, you should request a new DOI for the lesson. The steps are describe in the [How to Request a new DOI](https://github.com/programminghistorian/jekyll/wiki/How-to-Request-a-new-DOI) section of the Wiki. 
This part of the process should not take you more than one or two days, depending on the time difference you have with the UK (UTC +1).

**Comment by mdlincoln** [comment]
Just a note that you can request the DOI early in the PR review process - Sussex only needs to know which publication is making a new lesson, they don't need the finalized metadata at that stage - it's only after you merge the PR that you email them to say the new lesson is published and then activated. That way you can put in the DOI early on and attend to other errors like broken links.

However, n.b. the DOI doesn't turn into an active link that can be tweeted until Sussex responds that they've successfully submitted the metadata to CrossRef - make sense?

**Comment by rivaquiroga** [comment]
I'm doing the request process right now for a new lesson, and it seems that starting the request as step 2) might be better. 

----

## 2) Request DOI

Request a new DOI for the lesson following the steps described in the [Wiki](https://github.com/programminghistorian/jekyll/wiki/How-to-Request-a-new-DOI).
This part of the process should not take you more than one or two days, depending on the time difference you have with the UK (UTC +1).

**Comment by rivaquiroga** [comment]
@walshbr, what do do think about adding this update to https://github.com/programminghistorian/jekyll/pull/1844?

**Comment by walshbr** [comment]
That seems fine to me - you want pull requests to do a distinct thing, but it's also probably annoying to get multiple translation requests at the same time. We might as well combine them into one branch. If make the change on the branch that the pull request is using it should show up in the pull request feed. but let me know if it doesn't work. 

Also note that we're not merging that in for a little while because the PT team is working on those files currently.

**Comment by svmelton** [comment]
Sorry for missing this earlier, but it looks good to me!

**Comment by walshbr** [comment]
Oh whoops…since this was a separate issue from the one that was meant to take care of, I think we finalized #1844 without accounting for this text. So we'll need to open a fresh pull request for it, yes?

**Comment by rivaquiroga** [comment]
@walshbr, you are right. I totally forgot about this too.

**Comment by walshbr** [comment]
Are you still good with this text @rivaquiroga?

2) Request DOI
Request a new DOI for the lesson following the steps described in the Wiki.
This part of the process should not take you more than one or two days, depending on the time difference you have with the UK (UTC +1).

I can throw it up on a PR if so.

**Comment by spapastamkou** [comment]
Add a precision that the build will initially fail until the editor will have added the DOI?

**Comment by rivaquiroga** [comment]
@spapastamkou, that's a great idea.

**Comment by walshbr** [comment]
2. Request DOI
You need to request a new DOI for the lesson following the steps described in the Wiki.
This part of the process should not take you more than one or two days, depending on the time difference you have with the UK (UTC +1). You can start next steps while you wait, but note that builds will initially fail until the DOI has been added to the lesson metadata. 

How about that?

**Comment by rivaquiroga** [comment]
Are we ready to open the pull request?

@DanielAlvesLABDH, in case you weren’t following this issue, we are going to update the Editor Guidelines. We are adding one step to the Managing Editor checklist.

**Comment by DanielAlvesLABDH** [comment]
Thanks @rivaquiroga! I will make the changes in the Portuguese file

**Comment by walshbr** [comment]
Yeah sorry - I got distracted but can open the pull request shortly!
